python/ruff-ecosystem/ruff_ecosystem/main.py

```diff
@@ -53,7 +53,7 @@ async def limited_parallelism(coroutine: Awaitable[T]) -> T:
         async with semaphore:
             return await coroutine
 
-    comparisons: list[Exception | Comparison] = await asyncio.gather(
+    comparisons: list[BaseException | Comparison] = await asyncio.gather(
```

Comment:
Eek. Is this the type annotation for `asyncio.gather`?

Reply:
Yep, I think it's due to [asyncio.CancelledError being a subclass of BaseException, it previously was Exception though](https://docs.python.org/3/library/asyncio-exceptions.html#asyncio.CancelledError).